soccer/src/soccer/radio/sim_radio.cpp

```diff
@@ -94,17 +94,19 @@ SimRadio::SimRadio(bool blue_team)
         ip::udp::endpoint(address_, blue_team_ ? kSimBlueCommandPort : kSimYellowCommandPort);
     sim_control_endpoint_ = ip::udp::endpoint(address_, kSimCommandPort);
 
+    // assume robots 0-6 are always alive in sim
+    for (uint8_t robot_id = 0; robot_id < 7; robot_id++) {
```

Comment:
```suggestion
    for (uint8_t robot_id = 0; robot_id <= 6; robot_id++) {
``` is slightly more semantic